crates/cast/bin/cmd/run.rs

```diff
@@ -220,6 +227,19 @@ impl RunArgs {
             }
         };
 
+        if self.generate_local_signatures {
+            let project = config.project()?;
+            let compiler = ProjectCompiler::new().quiet(true);
```

Comment:
we don't need to actually compile here, right?
@klkvr is there an easier way to just get the artifacts?

Reply:
we need to compile as it might be run on a clean project. though I think we can only request abi here, should speed things up